src/js/controller/model.js

```diff
@@ -43,6 +44,11 @@ define([
             },
             _currentProvider = utils.noop;
 
+        if (config.cookies) {
```

Comment:
It's here https://github.com/jwplayer/jwplayer/pull/402/files#diff-50a064d2ccf19c5ee17fb7fc2e513525R7